draft-ietf-quic-multipath.md

```diff
@@ -667,23 +778,23 @@ the packet number alone would not guarantee the uniqueness of the nonce.
 
 In order to guarantee the uniqueness of the nonce, the nonce N is
 calculated by combining the packet protection IV with the packet number
-and with the least significant 32 bits of the Destination Connection ID
-sequence number.
+and with the least significant 32 bits of the path identifier pre-allocated 
+for the Destination Connection ID.
 
-{{Section 19 of QUIC-TRANSPORT}} encodes the Connection ID Sequence
-Number as a variable-length integer,
-allowing values up to 2^62-1; in this specification, a range of less than 2^32-1
+{{mp-new-conn-id-frame}} encodes the Path Identifier for Connection IDs 
+as a variable-length integer, allowing values up to 2^62-1; 
+in this specification, a range of less than 2^32-1
```

Comment:
I don't understand how a key update protects against nonce reuse. I'd prefer to define the path ID as a value between 0 and 2^32-1. 4 billion paths should be fine for most applications.

Of course, the varint format allows the encoding of up to 2^62-1, but this is something we can check immediately when decoding the transport parameter and the respective frames. There's plenty of precedent for limiting the range of acceptable values in RFC 9000.

Reply:
Reading through the issues, I think this is what @gloinul pointed out in #307.

Reply:
We should and can limit the maximum to 2^32-1. Older versions of Multipath draft has had that limitation implicitly, due to the number of CIDs being limited to 2^32, and the number of paths that can be opened being no more than the maximum number of CIDs.

Reply:
I think we have a consensus here that we need to limit the maximum of Path ID to 2^32-1 according to security consideration. It's updated in this PR.